factcast-schema-registry-cli/src/main/kotlin/org/factcast/schema/registry/cli/fs/FileSystemServiceImpl.kt

```diff
@@ -66,10 +69,38 @@ class FileSystemServiceImpl : FileSystemService {
         return Files.readAllBytes(file)
     }
 
+    override fun copyFromClasspath(source: String, target: Path) {
+        val url = javaClass.classLoader.getResource(source) ?: throw Exception("didnt found '$source' on classpath")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [TooGenericExceptionThrown - [url] at /src/factcast-schema-registry-cli/src/main/kotlin/org/factcast/schema/registry/cli/fs/FileSystemServiceImpl.kt:73:64](https://app.codacy.com/manual/uwe/factcast/pullRequest?prid=5032643)

Reply:
change to IOException